File: data/src/main/java/com/github/gmetal/data/entity/SpokenLanguageEntity.java
package com.github.gmetal.data.entity;

import com.bluelinelabs.logansquare.annotation.JsonField;
import com.bluelinelabs.logansquare.annotation.JsonObject;

@JsonObject
public class SpokenLanguageEntity {

    @JsonField
    public String iso6391;
    @JsonField
    public String name;
}

File: data/src/main/java/com/github/gmetal/data/entity/SpokenLanguageEntity.kt
package com.github.gmetal.data.entity

import com.bluelinelabs.logansquare.annotation.JsonField
import com.bluelinelabs.logansquare.annotation.JsonObject

@JsonObject
class SpokenLanguageEntity {

    @JsonField
    var iso6391: String? = null
    @JsonField
    var name: String? = null
}
